.github/workflows/common.yml

```diff
@@ -80,22 +80,16 @@ jobs:
       - name: Run go-sumtype
         run: bin/go-sumtype ./...
 
-      - name: Run golangci-lint
-        env:
-          COMMAND: "bin/golangci-lint run -c=.golangci-required.yml --out-format=line-number"
-          REDIRECT: "| bin/reviewdog -f=golangci-lint -reporter=github-pr-review -fail-on-error=true"
-          REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.ROBOT_TOKEN || secrets.GITHUB_TOKEN }}
-        run: |
-          if out=$( ${{ env.COMMAND }} ); exit_code=$?; [ $exit_code -ne 0 ]; then
-            if [ $exit_code -gt 1 ] || ${{ github.event.pull_request == null }}; then
-              echo "$out"
-              exit $exit_code
-            else
-              echo "$out" ${{ env.REDIRECT }}
-            fi
-          else
-            echo "$out"
-          fi
+      - name: golangci-lint
+        uses: reviewdog/action-golangci-lint@v2
+        with: 
+          github_token: ${{ secrets.ROBOT_TOKEN || secrets.GITHUB_TOKEN }}
+          go_version: "1.19"
+          reporter: github-pr-review
+          reviewdog_flags: "-f=golangci-lint"
+          fail_on_error: true
+          golangci_lint_flags: "-c=.golangci-required.yml --out-format=line-number"
+          golangci_lint_version: v1.46.2
```

Comment:
the current version that we use is 1.49.0
https://github.com/percona/pmm/blob/393d71c5be276fd683d0405938b8b4d35984ff4c/Makefile.include#L22

Reply:
Updated